src/js/components/CosmosErrorMessage.js

```diff
@@ -29,6 +29,11 @@ class CosmosErrorMessage extends React.Component {
       return this.appendRepositoryLink(message);
     }
 
+    // make "Package is already installed error" better
+    if (error.type === "PackageAlreadyInstalled") {
+      return "A service with the same name already exists. Try a different name.";
```

Comment:
I understand why this is necessary, we probably may want to open a ticket for the Cosmos team to brush some error messages.

Reply:
Talked with @MatApple offline, and since this is overriding the message of a specific `error.type`, we are fine doing this manual translation in this component.